hack/run_test

```diff
@@ -9,10 +9,20 @@
 set -exo nounset
 shopt -s nullglob
 
+TEST_LIST="\
+run_container_creation_tests \
+run_general_tests \
+run_change_password_test \
+run_replication_test \
+run_master_restart_test"
+
+test $# -eq 1 -a ${1-} == --list && echo $TEST_LIST && exit 0
 test -n "${IMAGE_NAME-}" || false 'make sure $IMAGE_NAME is defined'
```

Comment:
use 'echo "$TEST_LIST"' with the note above, so people can easily grep the output.